lib/common/src/Fonts.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace AmpProject;
+
+/**
+ * Class for fonts.
+ *
+ * @package ampproject/common
+ */
+final class Fonts
+{
+
+    const EMOJI_FONT_STACK = [
+        'Apple Color Emoji',
+        'Android Emoji',
+        'Segoe UI Emoji',
+        'Noto Color Emoji',
+        'EmojiSymbols',
+        'Symbola',
+        'Segoe UI Symbol',
+        'emoji',
+    ];
+
+    /**
+     * Get emoji font family property value.
+     *
+     * @return string Font-family property value.
+     */
+    public static function getEmojiFontFamilyValue()
+    {
+        return implode(
```

Comment:
This should be stored in a static variable, otherwise it is computed every single time the value is requested, even though it is purely static and logic is only for easy maintainability.

The usual construct for this is:
```php
static $fontFamilyValue = null;

if ($fontFamilyValue === null) {
	$fontFamilyValue = implode...;
}

return $fontFamilyValue;
```

Reply:
Thanks. Fixed in a4ed98330.